src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -181,4 +181,24 @@ public function committeeSupportAction(Request $request, CitizenProject $citizen
             'citizen_project' => $citizenProject,
         ]);
     }
+
+    /**
+     * @Route("/{slug}/acteurs", name="app_citizen_project_members_list_show")
+     * @Method("GET")
+     * @Security("is_granted('IS_AUTHENTICATED_FULLY')")
+     */
+    public function listActorsAction(CitizenProject $citizenProject): Response
+    {
+        $this->disableInProduction();
+
+        $citizenProjectManager = $this->get('app.citizen_project.manager');
```

Comment:
Please rebase, the service id does not exist anymore.

You can use Action injection because it was activated (see `services.xml`).